### ISSUE TITLE
Reconnection fails to restore service

### ISSUE THREAD

**Issue by jawi** [jawi/netmon#4]
The following is observed in the logs:
```
netmon[15053]: updating existing neighbour (10.10.0.1 <=> aa:bb:cc:dd:ee:ff), state = 4 vs 2)
netmon[15053]: disconnected from MQTT broker
netmon[15053]: reconnecting to MQTT broker
netmon[15053]: sending event on netmon.neigh :: {"last_seen":1563903467,"addr":"10.10.0.1","mac":"aa:bb:cc:dd:ee:ff","src":{"iface":"eth0","mac":"aa:bb:cc:dd:ee:ff","ip":"10.10.0.200"}}
netmon[15053]: failed to publish data to MQTT broker: The client is not currently connected.
netmon[15053]: updating existing neighbour (10.10.0.1 <=> aa:bb:cc:dd:ee:ff), state = 4 vs 4)
netmon[15053]: sending event on netmon.neigh :: {"last_seen":1563903502,"addr":"10.10.0.1","mac":"aa:bb:cc:dd:ee:ff","src":{"iface":"eth0","mac":"aa:bb:cc:dd:ee:ff","ip":"10.10.0.200"}}
```
The result is that though Netmon states it is pushing updates, in fact, no updates are coming through?!

**Comment by jawi** [comment]
Further investigation revealed that it seems to be related to a combination of `QoS = 1` and the use of Fluent-Bit as MQTT server.

**Comment by jawi** [comment]
Issue is fixed in Fluent-bit 1.3.0, which is released today.
